test/unittests/commonMocks.h

```diff
@@ -161,19 +161,18 @@ class NotifierMock : public Notifier {
          * actually created/sent */
     }
 
-    MOCK_METHOD6(sendNotifyContextRequest, void(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat));
+    MOCK_METHOD7(sendNotifyContextRequest, void(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, RenderFormat renderFormat, const std::vector<std::string>&  attrsFilter));
     MOCK_METHOD5(sendNotifyContextAvailabilityRequest, void(NotifyContextAvailabilityRequest* ncar, const std::string& url, const std::string& tenant, const std::string& fiwareCorrelator, Format f));
 
     /* Wrappers for parent methods (used in ON_CALL() defaults set in the constructor) */
-    void parent_sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat)
+    void parent_sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, RenderFormat renderFormat, const std::vector<std::string>&  attrsFilter)
     {
-      return Notifier::sendNotifyContextRequest(ncr, url, tenant, xauthToken, fiwareCorrelator, notifyFormat);
```

Comment:
But it is a fact that it worked with `return` until this PR, isn't it?

Assuming that, it is a *clever* idea to modify a code that we are not fully sure how it works (I think that I took that "parent wrapping" pattern from some place when I wrote the original code in this part... and in that place I guess that they were using `return`... for unknown reasons but maybe powerful enough)?

Not really sure what is better, anyway :)